doc/reference/rpc_command.rst

```diff
@@ -0,0 +1,1227 @@
+Commands
+========
+
+
+.. This document is generated via the `development:generate-documentation` command
+
+
+.. contents::
+   :depth: 2
+   :backlinks: none
+   :local:
+
+
+.. _StatusHandler:
+
+
+StatusHandler
+-------------
+
+
+.. _RpcCommand_GlobalDefinitionHandler_type:
+
+
+``type``
+""""""""
+
+
+**Default**: ``"formatted"``
+
+
+.. _FileInfoHandler:
+
+
+FileInfoHandler
+---------------
+
+
+.. _RpcCommand_GlobalDefinitionHandler_path:
```

Comment:
this is duplicated and is the reason for the lint failure: should be `FileInfoHandler`